bot.js

```diff
@@ -209,16 +209,18 @@ async function retrieveData(loc) {
         });
         console.log(`--> ${loc.name}: Data received @cab${index}\n\t> ${loc.cabs[index].newPlayers.toLocaleString()}`);
       }
-    }).catch((err) => {
-      console.log(err);
-      console.log('\n@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@\n--> Failed to retrieve data. @' + loc.name + '\n@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@\n');
     });
   }
 }
 
 // Updates player lists using new data
 function updatePlayerLists(loc) {
   loc.cabs.forEach(function(cab, index) {
+
+    // This is sort of bad and useless. It stops the bot from crashing if
+    // cab.players somehow reaches zero but also stops the cab from updating
+    // ever again. Also, it still doesn't take into account the case where
+    // cab.players.length is 1.
```

Comment:
I think I agree with this; it was residual from failed loads (which we now abort on) and/or just thinking that this list was after a fetch (0 becomes common).

Let's turn this into a commit to remove the functionality, and use the comment as the message/reasoning.